src/v2/Apps/Authentication/Server/checkForRedirect.ts

```diff
@@ -15,7 +16,7 @@ export const checkForRedirect = ({ req, res }) => {
   if (redirectTo === ("/reset_password" || "/user/delete")) {
     newRedirect = "/"
   } else {
-    newRedirect = redirectTo
+    newRedirect = sanitizeRedirect(redirectTo)
```

Comment:
use `sanitizeRedirect` only if `redirectTo` is defined. Otherwise we will get an error

<img width="1195" alt="error" src="https://user-images.githubusercontent.com/3513494/161991069-3250a4de-cb63-4d17-8249-8f7e9c6e1478.png">

Reply:
Great pickup! I was only testing _with_ the parameters 🙃  Very happy I tagged you to look at my changes. Will look at this after all hands

Reply:
Updated in [this](https://github.com/artsy/force/pull/9817/commits/c61250ca6c3554b2d675006a3e104f04bd9fed92) commit. Also found out that if you browse to `/login` as an already logged in user, it redirects you which is why it wasn't initially breaking for me.